sys/androidmedia/gstamcvideodec.c

```diff
@@ -490,6 +451,20 @@ create_src_caps (const GstAmcCodecInfo * codec_info, gboolean direct_rendering)
   return ret;
 }
 
+static GstFlowReturn
+gst_amc_video_dec_push_dummy (GstAmcVideoDec * self)
+{
+  GstBuffer *buf = gst_buffer_new ();
+  GstCaps *caps = gst_caps_new_simple ("video/x-amc", NULL);
+
+  gst_pad_set_caps (GST_VIDEO_DECODER (self)->srcpad, caps);
```

Comment:
This line is unacceptable: we are triggering new "set caps" proccess with each new buffer.

Reply:
yes, this line has to be resolved, it's just caused by copying code in a hurry. I did it actually..